### PR TITLE
CB-1157 Credential CRUD operations for AWS

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/user/UserPreferencesService.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import com.sequenceiq.cloudbreak.auth.security.authentication.AuthenticatedUserService;
+
+@Service
+public class UserPreferencesService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserPreferencesService.class);
+
+    @Inject
+    private UserPreferencesRepository userPreferencesRepository;
```
Comment (attilapalfi92) [inline environment/src/main/java/com/sequenceiq/environment/user/UserPreferencesService.java]: Next time use constructor injection. I will fix it in my pull request.

File: cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsCredentialView.java [hunk 2]
```diff
@@ -33,11 +53,43 @@ public Boolean isGovernmentCloudEnabled() {
     }
 
     public String getAccessKey() {
-        return cloudCredential.getParameter("accessKey", String.class);
+        String accessKey = cloudCredential.getStringParameter("accessKey");
```
Comment (attilapalfi92) [inline cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsCredentialView.java]: Using a `hasParameter` method in `CloudCredential` would be nicer, but I'll fix this.

Comment (topolyai5) [reply cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsCredentialView.java]: CloudCredentil has not hasParameters method

File: environment/src/main/java/com/sequenceiq/environment/user/UserPreferencesService.java (hunk 1 above)
Comment (topolyai5) [reply environment/src/main/java/com/sequenceiq/environment/user/UserPreferencesService.java]: It is not a project policy.

Comment (attilapalfi92) [reply environment/src/main/java/com/sequenceiq/environment/user/UserPreferencesService.java]: We talked about it, and we agreed to follow it in the environment service.

File: cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsCredentialView.java (hunk 2 above)
Comment (attilapalfi92) [reply cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsCredentialView.java]: Don't worry about it, I'll add it.

**Pull request merged**